### ISSUE TITLE
Explain creating a collada object

### ISSUE THREAD

**Issue by bagavi** [pycollada/pycollada#98]
Goal: I want to create a collada object/file with 5-10 cuboids in a 3-D space.

However, I am finding it very hard to follow the  documentation [here]( http://pycollada.github.io/creating.html) to create the above object. In particular, I am unable to understand the following:

1. What are `normal_floats` and `vert_floats` variables? What are the elements in these arrays?
2. What is `indices` variable? Here too, what do the elements mean?

I am teaching a class to a bunch of students and I was hoping to use this library for the project. Any help here would be appreciated. Thanks in advance!

**Comment by jterrace** [comment]
I think the docs are pretty good. I'd suggest you read up a bit on Collada itself:
http://www.wazim.com/Collada_Tutorial_1.htm
